### DISCUSSION TITLE
Github student developer pack thinks I'm not in my school

### DISCUSSION THREAD

**Discussion by hellothisisme123** [community/community#50385] [GitHub Education]
I'm trying to make my account use the github student developer pack so I can use all the different programs and things it has to offer. My school is in the program thing so that our students are able to use it but it's not working. I fill in the data, attach a photo and then it says. 

> Your application cannot be reviewed until you fix the following:
> 
> - The school you selected does not appear to have a campus location in your country. Your school-provided academic affiliation document must indicate virtual learning if you are remote.
> - You appear not to be near any campus location for the school you have selected. If you are a distance learner then your school-provided academic affiliation documentation must state so.

I am currently inside of my school trying to do this so my location is certainly close enough. My school email I'm using starts with student.myschool.org and before it said that emails using myschool.org are processed faster by I don't know what's causing this and I don't think that's the issue.

**Comment by samus-aran** [comment]
Hi @hellothisisme123 👋 

Here are some tips for when the application is viewing your location differently than expected:

- When asked, please enable the application to discover your location from the browser pop-up. Since settings for every browser are different, if you are having difficulty enabling us to see your location you should seek assistance from your browser's support pages.
  - Google provides this help article for Chrome browser users: https://support.google.com/chrome/answer/142065
  - Apple provides this help article for Safari browser users: https://support.apple.com/guide/mac-help/allow-apps-to-detect-the-location-of-your-mac-mh35873/mac
- If you are using a VPN, please disable it when you reapply. If you are not using a VPN, try reapplying when connected to the internet from a different network.
- If you are a distance learning student, please attach official, dated proof indicating that your program is a distance learning program or you are a distance learning student when you reapply.

Hope these help. 👍 Let us know how you get on.

**Reply by shekharbajgain1** [reply]
Please fix your github student verification. It seems github is incapable of handling. Yes, there are scammer but why punish the legit user? Currently, it is summer we can go out of states to visit and why is there such a horrific process. You should seriously fire the manager who gave such idea. 

When I try to verify I upload the i20 and nothing happens. The page submits and nothing happens. Microsoft ruined github.

**Reply by iam-rohan** [reply]
Yes same issue although I am at my campus the github education pack signup asks why am I not in campus. I mean I am there what am I suppose to do. I think this is really irresponsible to do such action. I know there can be scammers but hindering legit students from getting the allocated benefits is frustrating to see. Please help to fix this issue.

**Reply by davwheat** [reply]
> If you are a distance learning student, please attach official, dated proof indicating that your program is a distance learning program or you are a distance learning student when you reapply.

How can I attach evidence when the only option you provide me is using my webcam? That's really not a suitable way to attach a PDF from my laptop, is it?

**Reply by DanielZhang111** [reply]
It is absolutely absurd, I am in my home just 5 minutes walk from my school, but it does not verify and says "The school you selected does not appear to have a campus location in your country."! Anybody responsible;e for the student verification can reach out and help to solve this problem????

**Reply by DanielZhang111** [reply]
We have all evidence but the system is just denying. Where and who should we send the required documents to to get it verified? Thank you for any help!

**Reply by oneperfect-yj** [reply]
I previously got the student package, but now when I try to recertify, I can't pass the verification no matter what, even though I'm using the same or more certification materials. I am still currently a student.

**Reply by kozzins** [reply]
> Раньше я получал студенческий пакет, но теперь, когда я пытаюсь повторно пройти сертификацию, я не могу пройти проверку, несмотря ни на что, даже если я использую те же или несколько сертификационных материалов. В настоящее время я все еще студент.

do you solve this madness?

**Reply by T4rnover** [reply]
Dude, this thing is beyond belief broken.  Is there any managerial entity whatsoever that knows how bad this is?

**Comment by Shahnoor-Mahesar** [comment]
i have been facing the same issue while I am on my summer break my university is in another city, but the same country and i don't have any written proof from university about the semester break.
did you find the solution to this problem please share with me.

**Comment by HyHamza** [comment]
> I'm trying to make my account use the github student developer pack so I can use all the different programs and things it has to offer. My school is in the program thing so that our students are able to use it but it's not working. I fill in the data, attach a photo and then it says.
> 
> > Your application cannot be reviewed until you fix the following:
> > 
> > * The school you selected does not appear to have a campus location in your country. Your school-provided academic affiliation document must indicate virtual learning if you are remote.
> > * You appear not to be near any campus location for the school you have selected. If you are a distance learner then your school-provided academic affiliation documentation must state so.
> 
> I am currently inside of my school trying to do this so my location is certainly close enough. My school email I'm using starts with student.myschool.org and before it said that emails using myschool.org are processed faster by I don't know what's causing this and I don't think that's the issue.

The same problem here,
I am currently 2-3 kilometers away from my college, but still GitHub Education is not able to detect it and is asking why I am far from the campus. What is this? Now tell me what should I do? I am even showing my student ID, but the same problem persists. Tell me the solution, pleeease!

**Reply by talhayay11** [reply]
I am having the same problem here. Have you fixed it

**Comment by hellothisisme123** [comment]
lol silly github

**Comment by hellothisisme123** [comment]
No, i never did, i never got the student dev pack

**Comment by nhatnam2609** [comment]
same issues. But I'm in an internship. How to prove that?

**Comment by cccht** [comment]
But I'm in my campus...

**Comment by HyHamza** [comment]
No

**Comment by DN444** [comment]
Same issue. I am literally staying in my college hostel but Github can't recognize this.

**Comment by derrickzhuyz** [comment]
Same question. Moreover, although I use the university's internet within the campus without any VPN, "why not on campus" ridiculously persists and never stops.

**Reply by Luanrealcool** [reply]
check your region in window setting it should be the same with your college id if not change it

**Comment by venmidan** [comment]
Genuinely the worst system for validating student enrollment. I gave up after my 5th try - ON MY CAMPUS. Thanks Github.

**Comment by gvpofficial** [comment]
Even I have been facing the same issue

**Comment by jayrickaby** [comment]
This is the worst system ever. It asks me why I am so far on campus, and I send a picture FROM MY CAMERA of me at my campus and it will STILL be rejected. The name on my dated student ID is my PREFERRED name which does not appear on billing information, but the application does not allow "nicknames or variations". I have tried twice on my laptop, on college wifi on campus, and once on my phone hoping that it was just Linux being finicky with my location. what the hell is wrong with this???!
